### ISSUE TITLE
Update roadmap/rituals/agency parts of the README

### ISSUE THREAD

**Issue by filmaj** [adobe/developer.adobe.com#114]
The roadmap part for sure is out of date. @simonwex can you update the README? Also if you could do the same for the project milestones in this repo that would be super appreciated. We want to start using milestones to group issues together (for example, publishing the site, working out CI, etc.) and I wanted to use milestones for it, but there's lots in there, and I don't wanna mess up any setup you've already settled on.

**Comment by simonwex** [comment]
Absolutely. I've been doing a bit of thinking about this especially after our call yesterday.

**Comment by simonwex** [comment]
@mwbrooks I could use a thought partner on this, do you have any time today to chat?

**Comment by filmaj** [comment]
@simonwex is this a P1 or P2?
